src/components/pages/Wallet.tsx

```diff
@@ -140,18 +141,16 @@ const Invite: React.StatelessComponent<Props> = props => {
 
   return (
     <React.Fragment>
-      <Text style={[styles.inviteSectionText]}>
-        You have nothing to mint at this time.
-      </Text>
+      <Text style={[styles.inviteSectionText]}>Nothing to mint right now.</Text>
       <View>
         <Text style={styles.inviteSectionText}>
-          Invite a friend to earn 60 Raha:
+          Invite a friend to mint {REFERRAL_BONUS} Raha!
         </Text>
         <Button
           style={styles.button}
-          title="Invite"
+          title={`Invite +${REFERRAL_BONUS}`}
```

Comment:
We can actually make it have the full R 60.00, like how `MintButton` does it; you can do something like `title={["Invite +", referralBonus]}` where `referralBonus` is the actual Currency (with `CurrencyRole.None` to use the same color) object referring to that referral bonus. That makes it clearer it's an amount of currency and is also more consistent